src/Tribe/Views/V2/Views/Day_View.php

```diff
@@ -214,6 +215,47 @@ protected function sort_events( $events ) {
 
 	}
 
+	/**
+	 * Creates a HTML link and "fast forward" message to append to the "no events found" message.
+	 *
+	 * @param bool  $canonical         Whether to return the canonical (pretty) version of the URL or not.
+	 * @param array $passthru_vars     An optional array of query variables that should pass thru the method untouched
+	 *                                 in key and value.
+	 * @return string                  The html link and message.
+	 */
+	public function fast_forward( $canonical = false, array $passthru_vars = [] ) {
+		if ( ! empty( $this->context->get( 'tribe-bar-date' ) ) ) {
+			$date = $this->context->get( 'tribe-bar-date' );
+		} else if ( ! empty( $this->context->get( 'event_date' ) ) ) {
+			$date = $this->context->get( 'event_date' );
+		} else {
+			$date = strtok( $this->context->get( 'today' ), ' ' );
+		}
+
+		$cache_key = __METHOD__ . '_' . md5( wp_json_encode( array_merge( [ $date ] ) ) );
+
+		if ( isset( $this->cached_urls[ $cache_key ] ) ) {
+			return $this->cached_urls[ $cache_key ];
+		}
+
+		$next_event = (array) tribe_events()->where( 'starts_after', $date )->per_page(1)->get_ids();
```

Comment:
* :no_entry_sign: Expected 1 spaces after opening bracket; 0 found
* :no_entry_sign: Expected 1 spaces before closing bracket; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: 31b1c69fc5ea6c1f82b34108c7747b47</sub>